packages/devextreme/js/__internal/ui/date_box/m_date_box.strategy.native.ts

```diff
@@ -67,9 +67,12 @@ const NativeStrategy = DateBoxStrategy.inherit({
   },
 
   renderInputMinMax($input) {
+    const type = this.dateBox.option('type');
+    const format = type === 'datetime' ? 'yyyy-MM-ddTHH:mm' : 'yyyy-MM-dd';
```

Comment:
datebox also may have 'time' format, which leads to same issue, cover time case also, we can use map instead of ternary operator here

Reply:
```suggestion
    const format = {
     datetime: 'yyyy-MM-ddTHH:mm',
     date: 'yyyy-MM-dd',
     time: 'HH:mm',
    }[type];
```

Reply:
Also, I would prefer to consider the option where the format is not the expected value and, in this case, use the default value
Something like this:
`const defaultFormat = 'yyyy-MM-dd';
 
const format = {
	     datetime: 'yyyy-MM-ddTHH:mm',
       date: defaultFormat,
	     time: 'HH:mm',
	    }[type] ?? defaultFormat;`